RecoLocalTracker/SiPixelRecHits/plugins/gpuPixelRecHits.h

```diff
@@ -69,8 +70,8 @@ namespace gpuPixelRecHits {
 
 #ifdef GPU_DEBUG
     if (threadIdx.x == 0) {
-      auto k = first;
-      while (digis.moduleInd(k) == InvId)
+      auto k = clusters.moduleStart(1 + blockIdx.x);
```

Comment:
@VinInn some time ago you told me how to fix this...
I didn't do it at the time, so I hope I remembered it correctly.

Reply:
definitively NOT blockIdx.x: on CPU is always 1

Reply:
is this code before the change in cudaCompat?

Reply:
ok for auto k =